<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FORM</class>
 <widget class="QMainWindow" name="FORM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vermut Attack</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../usr/share/icons/JavaNunesMate48/bsd1.png</normaloff>../../../../../usr/share/icons/JavaNunesMate48/bsd1.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>windows/meterpreter/reverse_tcp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>windows/meterpreter/reverse_udp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>linux/meterpreter/reverse_tcp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>linux/meterpreter/reverse_udp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>macosx/meterpreter/reverse_tcp</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>localhost</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>192.168.0.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>168.168.1.6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200.204.0.249</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>49.88.112.66</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>124.238.113.126</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2000</string>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GERAR venon</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAvisos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>nada, nada, nada...</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMac">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MACADDRESS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineTextMacBSSID">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>00-00-00-00-00-00</string>
    </property>
    <property name="maxLength">
     <number>17</number>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoAtaque">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ATACA</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboPlacas">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>wlxc83a35cdb65d</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>wlp2s0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>wlan0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>run0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>enp1s0</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PLACA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butaoPrepare">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PREPARE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboCanal">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="buttonWifite">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Wifite Monitor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAirodump">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>AiroDump Monitor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="labelAvisosTextos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>A&amp;quela ?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
